packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -202,7 +202,16 @@ const main = async () => {
         )
         .addOption(new program.Option('--noHMR', 'disable the client-side hot module replacement'))
         .action(async ({inspect, noHMR}) => {
-            execSync(`node${inspect ? ' --inspect' : ''} ${resolvedSSRPath}`, {
+            // We use @babel/node instead of node because we want to support ES6 import syntax
+            const babelNode = p.join(
+                require.resolve('webpack'),
+                '..',
+                '..',
+                '..',
+                '.bin',
+                'babel-node'
+            )
+            execSync(`${babelNode} ${inspect ? '--inspect' : ''} ${resolvedSSRPath}`, {
```

Comment:
Why not just `npx babel-node`?

Reply:
i don't think require.resolve finds the binaries from `pwa-kit-dev` correctly, and that causes a delay in the start command when you run it the first time...